src/arguments.js

```diff
@@ -44,12 +44,14 @@ const unifyDescription = (description) => {
 const getDefaultValue = (description, regexp) => {
     let result = getValueByRegexp(description, regexp);
     if (result) {
-        result = removeCharAtBegin(result[5].trim(), ['\'', '\"', ',']);
-        result = removeCharAtTheEnd(result, ['\'', '\"', ',', '.']);
+        let extraChars = ['\'', '\"', ',', '['];
+        result = removeCharAtBegin(result[5].trim(), extraChars);
```

Comment:
Mb better use `removeChar` for `extraChars` and for remove dot use `removeCharAtTheEnd`